packages/contracts/src/EmailAccountRecovery.sol

```diff
@@ -71,8 +76,7 @@ abstract contract EmailAccountRecovery {
 
     /// @notice Completes the recovery process.
     /// @dev This function must be implemented by inheriting contracts to finalize the recovery process.
-    function completeRecovery() external virtual;
-
+    function completeRecovery(address account, bytes memory recoveryCalldata) external virtual;
```

Comment:
Haven't got the calldata construction in solidity working yet. Will aim to remove the `recoveryCalldata` arg if possible (currently unsure if it is possible). Should have an update on this over the weekend